components/ui-api-layer/internal/module/eventhandler.go

```diff
@@ -0,0 +1,104 @@
+package module
+
+import (
+	"github.com/golang/glog"
+	"github.com/kyma-project/kyma/components/ui-api-layer/pkg/apis/ui/v1alpha1"
+)
+
+type eventHandler struct {
+	module PluggableModule
+}
+
+func newEventHandler(module PluggableModule) *eventHandler {
+	return &eventHandler{module: module}
+}
+
+func (h *eventHandler) OnAdd(obj interface{}) {
+	resource, ok := obj.(*v1alpha1.BackendModule)
+	h.printIncorrectTypeErrorIfShould(ok, obj)
+
+	if !h.isAddOrDeleteEventRelatedToTheModule(resource) {
+		return
```

Comment:
What do you thing about adding some log with `warning` level in such case? Same in update event.

Reply:
Wait a second... Actually, there shouldn't be any warnings as other modules could handle the event. Every pluggable module will be wrapped with this function, and will ignore events which don't belong to them. So it's hard to log warnings only for events that have been missed, for example when there is no module with given name.